src/charm.py

```diff
@@ -123,6 +185,14 @@ def _on_ingress_revoked(self, event: IngressPerAppRevokedEvent) -> None:
         self._update_pebble_layer(event)
         self._update_login_ui_endpoint_relation_data(event)
 
+    @property
+    def _log_level(self) -> str:
+        return self.config["log_level"]
```

Comment:
In kratos and hydra operators I implemented a method that validates the user set log levels. It's called  _validate_config_log_level. Do you wanna add your implementation of something like this (or just copy paste the one in kratos or hydra) to _update_pebble_layer handler?

Reply:
given it is our application i would let that deal with the log level validation